www/front_src/src/Resources/Details/index.test.tsx

```diff
@@ -994,46 +991,6 @@ describe(Details, () => {
     );
   });
 
-  it('displays the shortcut links when the More icon is clicked', async () => {
```

Comment:
Why to remove the test about the icons ?

Reply:
because it was a test where you click on a more icon and then you check that the logs and reports icons are displayed correctly, according to the ticket the goal is to display the icons directly on the details panel, it is why i deleted the test 🤔

Reply:
Ok, isn't it necessary to check that the icons are present and that the links are still functional (in case we change the log and report link?)

Reply:
You must check that the link redirection

Reply:
ookk , I will add the test for link redirection (to ensure links are still functional)